whisper/whisperv5/peer.go

```diff
@@ -168,6 +196,20 @@ func (p *Peer) broadcast() error {
 	return nil
 }
 
+func (p *Peer) broadcastHashes(hashes []common.Hash) error {
+	if len(hashes) == 0 {
+		return nil
+	}
+	log.Trace("broadcast", "hashes", hashes)
+	if err := p2p.Send(p.ws, hashesCode, hashes); err != nil {
+		return err
+	}
+	for _, hash := range hashes {
+		p.advertised[hash] = struct{}{}
```

Comment:
Do you also plan to clean up the map after a while ?

Reply:
yes, this version was poc